src/test/scala/kubeyml/deployment/KubernetesStateYamlSpec.scala

```diff
@@ -128,37 +130,46 @@ class KubernetesStateYamlSpec extends FlatSpec with Matchers with ScalaCheckProp
                |          env:
                |            - name: "${envName}"
                |              value: "${envValue}"
-               |""".stripMargin).right.get
-          if (expectedYaml != deployment.asJson) {
-            println(expectedYaml.noSpaces)
+               |""".stripMargin)
+
+        val someYaml = expectedYaml match {
```

Comment:
Not sure what's going on here, please rebase with master and if you could try to follow the property testing paradigm. This class is only for sanitation, could you create your own to test your changes please?

Reply:
Ah ok, I think partly is my fault for not filtering out the rule on codacy for this.
But an unsafe right.get is acceptable in a test (for me) if we avoid more complex code.
In any case I have re-written this, sorry for the confusion

Reply:
Cool! I'll have a look.